pyerrors/correlators.py

```diff
@@ -277,19 +281,19 @@ def is_matrix_symmetric(self):
                         return False
         return True
 
-    def trace(self):
+    def trace(self) -> "Corr":
         """Calculates the per-timeslice trace of a correlator matrix."""
         if self.N == 1:
             raise ValueError("Only works for correlator matrices.")
-        newcontent = []
+        newcontent: list[Union[None, float]] = []
```

Comment:
Similar to Justus' question above: `newcontent` should be a `list of `None` or `Obs` (or `CObs`?), right?